frontend/package.json

```diff
@@ -48,6 +48,7 @@
         "ngx-owl-carousel-o": "14.0.1",
         "ngx-toastr": "15.2.2",
         "rxjs": "^7.5.6",
+        "three-dots": "^0.3.2",
```

Comment:
why do we need this library?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/56970649/1a3a0895-8868-4e24-b534-0e163d55e212)

for this nice animation of pending friendship request

Reply:
I believe you can do it by yourself, only using css